packages/browser-runtime-core/src/open-context-runtime.ts

```diff
@@ -48,17 +52,20 @@ export class OpenContextRuntime implements Runtime {
     return this.autocompleter.getCompletions(code);
   }
 
-  async evaluate(code: string): Promise<ShellResult> {
+  async evaluate(code: string): Promise<RuntimeEvaluationResult> {
     const evalFn = this.interpreter.evaluate.bind(this.interpreter);
-    return await this.shellEvaluator.customEval(
+    // Omitting rawValue before returning the result hence the unused variable
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { rawValue, ...result } = await this.shellEvaluator.customEval(
```

Comment:
What about just:

``` js
const  { type, printable, source } = await this.shellEvaluator.customEval(...);
return { type, printable, source };
```

We would make sure the contract is kept intact even if more properties are added to the original ShellApiResult. We also won't need a comment to understand the intention of the code to make `eslint` happy.

Reply:
Done in 1ab1e9d